added MediaTracker (#5)
awesome-jellyfin/awesome-jellyfin: README.md

  - [sonarr](https://github.com/Sonarr/Sonarr) Automates downloading & management of TV series [`🟣 TRaSH Guide`](https://trash-guides.info/Sonarr/)
  - [tdarr](https://github.com/HaveAGitGat/Tdarr) ` 📛 ` Distributed transcode automation + Library analytics + Video health checking
  - [recyclarr](https://github.com/recyclarr/recyclarr) Automatically sync TRaSH guides to Sonarr and Radarr instances

> - [MediaTracker](https://github.com/bonukai/MediaTracker) Media tracker and User rateing for Jellyfin and more via Plugin

Reviewer: ```suggestion
- [MediaTracker](https://github.com/bonukai/MediaTracker) Media tracker and user rating platform with [Jellyfin integration](https://github.com/bonukai/jellyfin-plugin-mediatracker)
```
